kolibri/plugins/learn/assets/src/views/content-card-grid/index.vue

```diff
@@ -32,8 +42,28 @@
 <script>
 
   import validateLinkObject from 'kolibri.utils.validateLinkObject';
+  import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+  import some from 'lodash/some';
+  import forEach from 'lodash/forEach';
+  import uiSelect from 'keen-ui/src/UiSelect';
   import contentCard from '../content-card';
+
   export default {
+    $trNameSpace: 'contentCardGrid',
+    $trs: {
+      display: 'Display',
+      all: 'All content ({ num, number })',
+      topics: 'Topics ({ count, number })',
+      exercises: 'Exercises ({ count, number })',
+      videos: 'Videos ({ count, number })',
+      audio: 'Audio ({ count, number })',
+      documents: 'Documents ({ count, number })',
+      html5: 'HTML5 Apps ({ count, number })',
+    },
```

Comment:
Sorry @christianmemije I meant on all of them to mirror how the strings were below previously.